.changeset/eight-cobras-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend': patch
+---
+
+Use `PermissionEvaluator` instead of `PermissionAuthorizer`.
```

Comment:
Worth calling out that `PermissionAuthorizer` is deprecated and will be removed in a future release?